applications/services/desktop/scenes/desktop_scene_pin_input.c

```diff
@@ -121,8 +133,19 @@ bool desktop_scene_pin_input_on_event(void* context, SceneManagerEvent event) {
         case DesktopPinInputEventResetWrongPinLabel:
             desktop_scene_locked_light_red(false);
             desktop_view_pin_input_set_label_primary(desktop->pin_input_view, 0, 0, NULL);
-            desktop_view_pin_input_set_label_secondary(
-                desktop->pin_input_view, 44, 25, "Enter PIN:");
+            uint32_t attempts = furi_hal_rtc_get_pin_fails();
+            state->enter_pin_string = furi_string_alloc();
+            furi_string_set_str(state->enter_pin_string, "Enter PIN (Fails: ");
+            furi_string_cat_printf(state->enter_pin_string, "%lu", attempts);
+            furi_string_cat_str(state->enter_pin_string, "):");
+            const char* enter_pin_cstr = furi_string_get_cstr(state->enter_pin_string);
```

Comment:
You only use the var `enter_pin_cstr` in the if block, so it should be declared within there.

Reply:
Thanks for the infos! Will probably look at it tommorow :)